### PR TITLE
Analytic report active user

### PR DESCRIPTION
**Pull request by carmenlau**
- Breaking change: Update `analytic report` command with period flag and it is required. The commands will be like this,
        - `go run ./cmd/portal analytic report user --period=last-week`
        - `go run ./cmd/portal analytic report project --period=last-week`
        - `go run ./cmd/portal analytic report project --period=last-month`
- Add active users count to project weekly and monthly report

### CODE REVIEW

File: user-weekly-report.csv [hunk 1]
```diff
@@ -0,0 +1,2 @@
+Year,Week,Range From,Range To,Number of new users,Number of new users who have created project
```
Comment (louischan-oursky) [inline user-weekly-report.csv:1]: Is this file committed by mistake?

File: cmd/portal/analytic_report.go [hunk 2]
```diff
@@ -146,14 +124,24 @@ var cmdAnalyticReport = &cobra.Command{
...
+		}
+
 		var data *analyticlib.ReportData
 		dbPool := db.NewPool()
 		switch reportType {
 		case analytic.ReportTypeUserWeeklyReport:
-			year, week, err := getISOWeek()
-			if err != nil {
-				return err
+			if periodicalType != periodical.Weekly {
```
Comment (louischan-oursky) [inline cmd/portal/analytic_report.go:141]: I have another idea of  changing the CLI interface.

```sh
go run ./cmd/portal analytic report user --period=last-week
go run ./cmd/portal analytic report project --period=last-month
```

Then we eliminate the problem of duplicating the period in the report type argument.

File: pkg/lib/feature/verification/store_pq.go [hunk 3]
```diff
@@ -35,7 +35,7 @@ func (s *StorePQ) scan(scn db.Scanner) (*Claim, error) {
 		&c.CreatedAt,
 	)
 	if errors.Is(err, sql.ErrNoRows) {
-		return nil, ErrClaimUnverified
+		return nil, nil
```
Comment (louischan-oursky) [inline pkg/lib/feature/verification/store_pq.go:38]: The value of `ErrClaimUnverified` is used in VerificationService. It seems that we have to return ErrClaimUnverified to signify the claim is not verified (or not found in the verified claim table).

Comment (carmenlau) [reply pkg/lib/feature/verification/store_pq.go]: Sorry, updated it accidentally, removed it:(

File: cmd/portal/analytic_report.go (hunk 2 above)
Comment (carmenlau) [reply cmd/portal/analytic_report.go:141]: Updated!

**Comment by louischan-oursky** [comment]
Conflicted :(

**Comment by carmenlau** [comment]
Rebased!

**Pull request merged**